packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -408,9 +499,18 @@ def imshow(
     else:
         raise ValueError(
             "px.imshow only accepts 2D single-channel, RGB or RGBA images. "
-            "An image of shape %s was provided" % str(img.shape)
+            "An image of shape %s was provided."
+            "Alternatively, 3-D single or multichannel datasets can be"
+            "visualized using the `facet_col` or `animation_frame` arguments."
+            % str(img.shape)
         )
 
+    # Now build figure
+    col_labels = []
+    if facet_col is not None:
+        slice_label = "slice" if labels.get("facet") is None else labels["facet"]
```

Comment:
the PX default behaviour here is to have the label be called `facet_col` and have it be overrideable via `labels["facet_col"]` OR to have it be the value of `facet_col` (i.e. "day of week") and have it be overrideable via `labels["day of week"]`

Reply:
> OR to have it be the value of facet_col (i.e. "day of week") and have it be overrideable via labels["day of week"]

we can maybe leave this out for now/do it later... `px.imshow` already doesn't do this for x/y/color and it mostly only applies in the case of xarray.

Reply:
yes exactly here we don't have column names, and the case of xarrays is handled correctly I think. So I can leave it like this?